R/fix_c3_costs.R

```diff
@@ -0,0 +1,34 @@
+#' Apply costs fix to specialty C3 in acute processing.
+#'
+#' @param data Acute extract file after processing
+#' @param year Financial year e.g. 1819
+#'
+#' @return A data frame with costs applied
+#' @export
+#'
+#'@importFrom dplyr mutate case_when
+fix_c3_costs <- function(data, year) {
+  if (year >= "1819") {
+    # Amend cost total net
```

Comment:
`year` should be a character or numeric. You've used a numeric in the script but are comparing against a character here. This could cause unintended issues, I think R will convert to a character to be able to compare them...
```suggestion
fix_c3_costs <- function(data, year) {
if (as.integer(first_part) + 1L != as.integer(second_part)) {
    stop("Year has been entered in the wrong format, try again using form `1718` or use function `convert_year_to_fyyear` to convert to the financial year form.")
}

  if (year >= "1819") {
    # Amend cost total net
```